### PR TITLE
Noetic gazebo classic

### PR DESCRIPTION
**Pull request by Guillaumebeuzeboc**
**DO NOT MERGE**
This is here only for review.

These are the changes suggested to build a snap for ROS noetic compatible with the Gazebo simulation.
The main differences are:
- use of the snap daemons functionalities for map saving and others
- maps are now saved in `SNAP_USER_COMMON` (user being root in snap daemons)

The corresponding MR for the snap is available [here](https://github.com/canonical/turtlebot3c/pull/1)

**DO NOT MERGE**

### CODE REVIEW

**Comment by Guillaumebeuzeboc** [comment]
@artivis I do not have enough privilege on this repo to assign you for review

**Comment by Guillaumebeuzeboc** [comment]
@artivis updated the branch to contain the following changes:
- map is saved in "map" subdir
- armhf architecture
- joy application (tested)
- allow the snap to run on a robot or with the simulation (with the simulation parameter). I could test that the robot core was starting.

File: README.md [hunk 1]
```diff
@@ -4,29 +4,34 @@
 
 `turtlebot3c` packaged as a snap.  
 `turtlebot3c` is a collection of `launchfiles` and configuration files to ease working with the Turtlebot3.
-The snap is meant to be run on the robot, allowing one to quickly get up and running with a self contained Turtlebot3.
+The snap is meant to be run with the simulation, allowing one to quickly get an up and running Turtlebot3 application.
```
Comment (artivis) [inline README.md]: Please update.

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -1,13 +1,26 @@
 name: turtlebot3c
-version: '0.1'
+version: git
+license: GPL-3.0
 summary: Turtlebot3c core snap
 description: |
   This snap automatically spawn a roscore and the core components for the
   Turtlebot3.
 
+  This snap is meant to run on the Turtlebot 3 robot or with the simulation
```
Comment (artivis) [inline snap/snapcraft.yaml:9]: Update please.

Comment (Guillaumebeuzeboc) [reply snap/snapcraft.yaml:9]: What must be updated?

**Comment by Guillaumebeuzeboc** [comment]
Closed in favor of https://github.com/canonical/turtlebot3c-snap/pull/2

**Pull request closed without merging**